Patch by ecopunk so fix FTBFS on some systems with vala-0.16
--- src/Makefile.am.orig    2012-03-13 13:20:12.000000000 -0600
+++ src/Makefile.am 2012-04-08 01:42:33.147393602 -0600
@@ -34,6 +34,10 @@
    -DINDICATORDIR=\""$(INDICATORDIR)"\"

 unity_greeter_VALAFLAGS = \
+   --vapidir . \
+   --pkg config \
+   --pkg fixes \
+   --pkg indicator \
    --pkg posix \
    --pkg gtk+-3.0 \
    --pkg gdk-x11-3.0 \
